### PR TITLE
Update notifiers doc, add auto inclusion doc

### PR DESCRIPTION
**Pull request by scottcwilson**
I ran out of gas writing the auto inclusion doc but I have meant to do this for a while.  Please throw out suggestions. 
At a minimum this is a start which can be built out over time.

### CODE REVIEW

File: content/dev/code/inclusion.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+- A storefront side page will automatically load its own language file.  So the page `includes/modules/pages/contact_us/` will load `includes/languages/english/contact_us.php`
+- All files in `includes/languages/english/extra_definitions` are automatically loaded.
+
+### Extra Datafiles and Extra Configures 
+
+FIXME need content
+
+### Extra Cart Actions 
+
+FIXME need content
```
Comment (drbyte) [inline content/dev/code/inclusion.md:24]: These are identical to the `extra_functions` dir: every *.php file here is loaded automatically.

File: content/dev/code/notifiers.md [hunk 2]
```diff
@@ -189,21 +194,27 @@ If your observer-class performs actions *prior to* the page-specific loading, e.
 
 ### Auto-loaded Observers
 
-If you're developing a plugin that uses an observer-class, you normally have to provide ***two*** files in your plugin's distribution to get that class loaded and instantiated:
+If you're developing a plugin that uses an observer-class, you would traditionally have to provide ***two*** files in your plugin's distribution to get that class loaded and instantiated:
```
Comment (drbyte) [inline content/dev/code/notifiers.md:197]: I'm not sure `traditionally` is appropriate here.
`you would provide` is fine.

File: content/dev/code/notifiers.md [hunk 3]
```diff
@@ -189,21 +194,27 @@ If your observer-class performs actions *prior to* the page-specific loading, e.
...
 
 1. /includes/auto_loaders/config.your_plugin.php
 2. /includes/classes/observers/class.your_plugin.php
 
-Starting with Zen Cart v1.5.3, built-in functionality will do the  "heavy lifting" to get your class-file loaded and instantiated &mdash; so  long as your class doesn't have any special requirements on its load  point (auto-loaded classes are loaded at point `175`, after all other  system dependencies are loaded).   Here are the requirements (as pulled  from the file `/includes/init_includes/init_observers.php`):
+Starting with Zen Cart v1.5.3, built-in functionality will do the  "heavy lifting" to get your class-file loaded and instantiated, saving you some effort.
+
+Here are the requirements (as pulled  from the file `/includes/init_includes/init_observers.php`):
```
Comment (drbyte) [inline content/dev/code/notifiers.md:204]: The stuff in parentheses could be dropped.

File: content/dev/code/inclusion.md [hunk 4]
```diff
@@ -8,7 +8,11 @@ weight: 10
 To reduce the number of changes required for plugin authors, Zen Cart does some automatic inclusion of files. 
 
 ### Functions 
+<<<<<<< HEAD
+Any `*.php` files placed in `includes/functions/extra_functions` are loaded automatically, and can be called by any storefront side logic.  Similarly, on the admin side, functions in `admin/includes/functions/extra_functions` are automatically loaded. 
+=======
 Any functions placed in `includes/functions/extra_functions` are loaded automatically, and can be called by any storefront side logic.  Similarly, on the admin side, functions in `admin/includes/functions/extra_functions` are automatically loaded. 
+>>>>>>> fe0f67393dbd6c28d2cd1a325c21e64b1f2ae9ba
```
Comment (drbyte) [inline content/dev/code/inclusion.md]: Oops - some merge conflict here

Comment (scottcwilson) [reply content/dev/code/inclusion.md]: got it - will fix.

**Pull request merged**